index.html

```diff
@@ -5,11 +5,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+
+    <style>
+        a{
+            margin: 0.5rem;
+        }
+    </style>
```

Comment:
I think it's better to use  external CSS

Reply:
Hey, that is just temporary. I have to design the home page.